test/smtcallback.js

```diff
@@ -66,20 +66,28 @@ tape('SMTCheckerCallback', function (t) {
       sources: input
     });
 
-    // Solidity 0.6.9 comes with z3
     var tests;
     if (!semver.gt(solc.semver(), '0.6.8')) {
+      // Up to version 0.6.8 there were no embedded solvers.
```

Comment:
Oh it's "not greater than" - easy as cake!

Reply:
Should we change these to `lte`?

Reply:
Prolly